test/e2e_test.go

```diff
@@ -64,7 +64,7 @@ var tests = map[string]testInput{
 	},
 }
 
-func TestMain(t *testing.T) {
+func Test_Main(t *testing.T) {
```

Comment:
The go convention is camelcase
```suggestion
func TestMain(t *testing.T) {
```

Reply:
I'm not sure why but I can't run `TestMain` in my IDE. It's expecting a different function signature.

Reply:
Changed name

Reply:
Changed name